src/Admin/BaseBlockAdmin.php

```diff
@@ -107,14 +96,8 @@ public function prePersist($object): void
         if ($object->getPage() instanceof PageInterface) {
             $object->getPage()->setEdited(true);
         }
-
-        // fix weird bug with setter object not being call
```

Comment:
We can remove this code ? There is a comment talking about a bug

Reply:
there is also #1291